packages/express-list-endpoints/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Portions copyright (c) 2015 Alberto Fernandez Medina
+Portions copyright (c) 2024 PrairieLearn, Inc.
```

Comment:
This originally read:

> Copyright (c) 2015 Alberto Fernandez Medina

From some Googling, it looks like this is an acceptable way to fork MIT-licensed code. Honestly, this is my first time doing this, so it'd be good to get a second opinion.

Reply:
Haha, I just replied to the earlier comment above with exactly the same conclusion.